docs/preliminaries.adoc

```diff
@@ -7,9 +7,34 @@ To use *Arquillian Cube* you need a _Docker_ daemon running on a computer (it ca
 By default the _Docker_ server uses UNIX sockets for communicating with the _Docker_ client. *Arquillian Cube* will attempt to detect the operating system it is running on and either set _docker-java_ to use UNIX socket on _Linux_ or to <<Boot2Docker>> on _Windows_/_Mac_ as the default URI.
 
 If you want to use TCP/IP to connect to the Docker server, you'll need to make sure that your _Docker_ server is listening on TCP port.
-To allow _Docker_ server to use TCP add the following line to +/etc/default/docker+:
+To allow _Docker_ server to use TCP set the _Docker daemon options_, the exact process for which varies by the way you
+launch the Docker daemon and/or the underlying OS:
 
-+DOCKER_OPTS="-H tcp://127.0.0.1:2375 -H unix:///var/run/docker.sock"+
+* systemd (Ubuntu, Debian, RHEL 7, CentOS 7, Fedora, Archlinux) — edit docker.service and change the ExecStart value
```

Comment:
Can you explicitly mention here Fedora - docker community edition(`docker ce`)? So it will help for user to understand why respective docker installation directory is changing